client/components/Dashboard.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useState, useEffect } from 'react';
 import { useQuery, gql } from '@apollo/client';
-import Logger from './Logger';
+// import Logger from './Logger';
```

Comment:
Could we rename the `logger` component to something like `chart` or `display`? I find logger to confusing with the logger repository.

Reply:
Right now I am still deciding whether I will use logger or querries component. If I choose logger, I will rename it to whatever you guys decide on.